### ISSUE TITLE
bash installer wants ruamel_yaml

### ISSUE THREAD

**Issue by loriab** [conda/constructor#33]
I built a fairly plain installer (no custom channels or `packages` section) with conda `4.1.3` and constructor `1.2.0` _or_ `1.2.1`. It builds fine. But when I run the installer (in either a conda-free or conda-and-ruamel_yaml-available shell), I get the below. An installer built with constructor `1.2.0` and conda `4.0.9` builds and installs fine.
```
>>> bash shortened/psi4conda-0.9.0-Linux-x86_64.sh 

Welcome to psi4conda 0.9.0

psi4conda will now be installed into this location:
/shortened/psi4conda

  - Press ENTER to confirm the location
  - Press CTRL-C to abort the installation
  - Or specify a different location below

[/shortened/psi4conda] >>> /shortened/psi4conda90b
PREFIX=/shortened/psi4conda90b
installing: python-2.7.11-0 ...
Python 2.7.11 :: Continuum Analytics, Inc.
installing: alabaster-0.7.8-py27_0 ...
installing: babel-2.3.3-py27_0 ...
installing: cmake-3.3.1-0 ...
installing: conda-4.1.3-py27_0 ...
ERROR:root:code for hash md5 was not found.
Traceback (most recent call last):
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type md5
ERROR:root:code for hash sha1 was not found.
Traceback (most recent call last):
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha1
ERROR:root:code for hash sha224 was not found.
Traceback (most recent call last):
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha224
ERROR:root:code for hash sha256 was not found.
Traceback (most recent call last):
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha256
ERROR:root:code for hash sha384 was not found.
Traceback (most recent call last):
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha384
ERROR:root:code for hash sha512 was not found.
Traceback (most recent call last):
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/shortened/psi4conda90b/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha512
No yaml library available.
To proceed, please conda install ruamel_yaml
```

At installer build time, I tried explicitly adding `ruamel_yaml` to the `specs` list along with `install_in_dependency_order: true`, but that only yielded:
```
>>> constructor installer/
platform: linux-64
Fetching package metadata .....
Solving package specifications: ..........
Traceback (most recent call last):
  File "/shortened/miniconda11/bin/constructor", line 6, in <module>
    sys.exit(main())
  File "/shortened/miniconda11/lib/python2.7/site-packages/constructor/main.py", line 146, in main
    verbose=opts.verbose)
  File "/shortened/miniconda11/lib/python2.7/site-packages/constructor/main.py", line 81, in main_build
    fcp.main(info, verbose=verbose)
  File "/shortened/miniconda11/lib/python2.7/site-packages/constructor/fcp.py", line 172, in main
    resolve(info)
  File "/shortened/miniconda11/lib/python2.7/site-packages/constructor/fcp.py", line 41, in resolve
    dists.extend(d + '.tar.bz2' for d in r.graph_sort(sort_info))
AttributeError: 'Resolve' object has no attribute 'graph_sort'
```

**Comment by ilanschnell** [comment]
This should be fix in 1.3.0, which as released today.  Please confirm.

**Comment by loriab** [comment]
Confirmed, all fixed, thanks!
